R/endpoints.R

```diff
@@ -4,6 +4,8 @@ api_build <- function(queue) {
             serializer = serializer_json_hintr())
   pr$handle("POST", "/validate/baseline", endpoint_validate_baseline,
             serializer = serializer_json_hintr())
+  pr$handle("POST", "/validate/survey_and_programme", endpoint_validate_survey_programme,
```

Comment:
Previously/elsewhere we had decided that dashes were REST API best practice

Reply:
Yep, had forgotten about that. That's updated now.